app/src/components/games/reefscape/ReefscapeAutoModal.tsx

```diff
@@ -0,0 +1,438 @@
+import {Modal, Pressable, Text, View} from 'react-native';
+import React, {useMemo, useState} from 'react';
+import {useTheme} from '@react-navigation/native';
+import Svg, {Path} from 'react-native-svg';
+import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
+import {ReefscapeField} from './ReefscapeField';
+import {AutoPath} from './AutoPath';
+import {
+  ReefscapeActionIcon,
+  ReefscapeActions,
+  ReefscapeActionType,
+} from './ReefscapeActions';
+import {ReefscapeLevels} from './ReefscapeLevels';
+
+interface HistoryAction {
+  action: string;
+  pieceID: number;
+}
+
+interface LinkItemMap {
+  [key: string]: {
+    value: number;
+    index: number;
+  };
+}
+
+const ActionButton = ({
+  positiveAction,
+  negativeAction,
+  goodLabel,
+  badLabel,
+  flex,
```

Comment:
why is flex passed in as a parameter? just directly hardcode flex:1/2 in the component